### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by Arthusppp**
Update Polish translation

### CODE REVIEW

File: files/lang/pl.po [hunk 1]
```diff
@@ -574,31 +572,23 @@ msgstr "%{spell} zadaje %{damage} pkt. obrażeń żywej istocie."
 msgid "The %{spell} does %{damage} damage to all living creatures."
 msgstr "%{spell} zadaje %{damage} pkt. obrażeń każdej żywej istocie."
 
-#, fuzzy
 msgid "The %{attacker}' attack blinds the %{target}!"
-msgstr "Atak jednorożców oślepia jednostkę (%{name})!"
+msgstr "Atak %{attacker} oślepia %{name}!"
```
Comment (michkry) [inline files/lang/pl.po:576]: It should be {target} instead of {name}

File: files/lang/pl.po [hunk 2]
```diff
@@ -574,31 +572,23 @@ msgstr "%{spell} zadaje %{damage} pkt. obrażeń żywej istocie."
...
 
-#, fuzzy
 msgid "The %{attacker}' gaze turns the %{target} to stone!"
-msgstr "Spojrzenie meduz zamienia jednostkę (%{name}) w kamień!"
+msgstr "Spojrzenie %{attacker} zamienia %{name} w kamień!"
```
Comment (michkry) [inline files/lang/pl.po:579]: It should be {target} instead of {name}

File: files/lang/pl.po [hunk 3]
```diff
@@ -574,31 +572,23 @@ msgstr "%{spell} zadaje %{damage} pkt. obrażeń żywej istocie."
...
 
-#, fuzzy
 msgid "The %{attacker}' curse falls upon the %{target}!"
-msgstr "Ta jednostka (%{name}) zostaje dotknięta klątwą mumii!"
+msgstr "%{attacker} przeklina %{name}!"
 
-#, fuzzy
 msgid "The %{target} are paralyzed by the %{attacker}!"
-msgstr "Atak cyklopów paraliżuje jednostkę (%{name})!"
+msgstr "Atak %{attacker} paraliżuje %{name}!"
```
Comment (michkry) [inline files/lang/pl.po:585]: I think that here we should keep the original meaning of the sentence. I propose: %{target} zostal paraliżowany przez %{attacker}!

File: files/lang/pl.po [hunk 4]
```diff
@@ -2691,36 +2675,37 @@ msgstr "Wybierz rozdzielczość gry:"
 msgid "Change resolution of the game."
 msgstr "Zmień rozdzielczość gry."
 
-#, fuzzy
 msgid "Change audio settings of the game."
-msgstr "Wybierz język, rozdzielczość i ustawienia gry."
+msgstr "Zmień ustawienia audio gry."
```
Comment (michkry) [inline files/lang/pl.po:2679]: We may consider using "dźwiękowe" instead of "audio".

File: files/lang/pl.po [hunk 5]
```diff
@@ -2691,36 +2675,37 @@ msgstr "Wybierz rozdzielczość gry:"
...
 "move as smoothly as black and white ones."
 msgstr ""
+"Zmień kursor na kolorowy lub czarno-biały. Kolorowy kursor wygląda ładniej, "
+"ale czasem nie jest tak płynny jak czarno-biały."
 
 msgid ""
 "Toggle text support mode to output extra information about windows and "
 "events in the game."
 msgstr ""
+"Włącza lub wyłącza dodatkowe informacje w oknach lub o zdarzeniach w grze."
```
Comment (michkry) [inline files/lang/pl.po:2704]: We may consider the following translation:
"Włącza lub wyłącza dodatkowe informacje o oknach i zdarzeniach w grze."

File: files/lang/pl.po [hunk 6]
```diff
@@ -3175,20 +3160,20 @@ msgid "Interface"
 msgstr "Intefejs"
 
 msgid "Change the speed at which your heroes move on the main screen."
-msgstr "Zmienia szybkość, z jaką poruszają się Twoi bohaterowie na ekranie."
+msgstr "Zmień szybkość, z jaką poruszają się Twoi bohaterowie na ekranie."
```
Comment (michkry) [inline files/lang/pl.po:3163]: We should change "Twoi" to "twoi".

File: files/lang/pl.po [hunk 7]
```diff
@@ -3493,11 +3474,11 @@ msgstr "fheroes2 Resurrection Team prezentuje"
 msgid "Greetings!"
 msgstr "Witaj!"
 
-#, fuzzy
 msgid ""
 "Welcome to Heroes of Might and Magic II powered by fheroes2 engine! Before "
 "starting the game please choose game resolution."
-msgstr "fheroes2 wita! Przed rozpoczęciem gry wybierz rozdzielczość gry."
+msgstr "Witamy w grze Heroes of Might and Magic II na silniku fheroes2! "
+"Przed rozpoczęciem gry wybierz rozdzielczość gry."
```
Comment (michkry) [inline files/lang/pl.po:3481]: We may consider changing "Przed rozpoczęciem gry wybierz rozdzielczość gry." to "Przed rozpoczęciem gry wybierz rozdzielczość ekranu." or "Przed rozpoczęciem gry wybierz rozdzielczość.". I think the last word "gry" is redundant.

File: files/lang/pl.po [hunk 8]
```diff
@@ -8381,11 +8359,11 @@ msgstr "Mumia"
 msgid "Spell Scroll"
 msgstr "Zwój z zaklęciem"
 
-#, fuzzy
 msgid ""
 "This %{name} gives your hero the ability to cast the %{spell} spell if your "
 "hero has a Magic Book."
-msgstr "%{name} daje twojemu bohaterowi mozliwość rzucania czaru %{spell}"
+msgstr "Ten %{name} daje twojemu bohaterowi mozliwość rzucania czaru %{spell} "
+"nawet jeśli bohater nie ma księgi zaklęć."
```
Comment (michkry) [inline files/lang/pl.po:8366]: In original text, the hero must have a Magic Book in order to cast the spell. The translation says it is not necessary.

File: files/lang/pl.po (hunk 1 above)
Comment (Arthusppp) [reply files/lang/pl.po:576]: Yes! Corrected.

File: files/lang/pl.po (hunk 2 above)
Comment (Arthusppp) [reply files/lang/pl.po:579]: Yes! Corrected.

File: files/lang/pl.po (hunk 3 above)
Comment (Arthusppp) [reply files/lang/pl.po:585]: Ok.

File: files/lang/pl.po [hunk 9]
```diff
@@ -574,31 +572,23 @@ msgstr "%{spell} zadaje %{damage} pkt. obrażeń żywej istocie."
...
 msgid "The %{target} are paralyzed by the %{attacker}!"
-msgstr "Atak cyklopów paraliżuje jednostkę (%{name})!"
+msgstr "Atak %{attacker} paraliżuje %{name}!"
 
-#, fuzzy
 msgid "The %{attacker} dispel all good spells on your %{target}!"
-msgstr ""
-"Arcymagowie rozpraszają wszystkie korzystne zaklęcia u tej jednostki "
-"(%{name})!"
+msgstr "%{attacker} rozpraszają wszystkie korzystne zaklęcia u %{name}!"
```
[the remark this answers is not in the document]
Comment (Arthusppp) [reply files/lang/pl.po:588]: Yes! Updated.

File: files/lang/pl.po [hunk 10]
```diff
@@ -2565,20 +2553,19 @@ msgid "Music Type"
...
+msgstr "Zmień poziom głośności efektów."
 
 msgid "Change the type of music."
-msgstr "Zmienia rodzaj muzyki."
+msgstr "Zmień rodzaj muzyki."
 
-#, fuzzy
 msgid "Toggle 3D effects of foreground sounds."
-msgstr "Zmiana poziomu głośności efektów."
+msgstr "Zmień efekt 3D dla pierwszoplanowych dźwięków."
```
[the remark this answers is not in the document]
Comment (Arthusppp) [reply files/lang/pl.po:2568]: Ok.

File: files/lang/pl.po (hunk 4 above)
Comment (Arthusppp) [reply files/lang/pl.po:2679]: Ok.

File: files/lang/pl.po (hunk 5 above)
Comment (Arthusppp) [reply files/lang/pl.po:2704]: Ok!

File: files/lang/pl.po (hunk 6 above)
Comment (Arthusppp) [reply files/lang/pl.po:3163]: Prawda! I usuwam przecinek (bo tutaj nie mamy zdań podrzędnych).

File: files/lang/pl.po (hunk 7 above)
Comment (Arthusppp) [reply files/lang/pl.po:3481]: Totally agree!

File: files/lang/pl.po (hunk 8 above)
Comment (Arthusppp) [reply files/lang/pl.po:8366]: You are right, corrected!

File: files/lang/pl.po [hunk 11]
```diff
@@ -574,31 +572,23 @@ msgstr "%{spell} zadaje %{damage} pkt. obrażeń żywej istocie."
...
 
-#, fuzzy
 msgid "The %{attacker}' gaze turns the %{target} to stone!"
-msgstr "Spojrzenie meduz zamienia jednostkę (%{name}) w kamień!"
+msgstr "Spojrzenie %{attacker} zamienia %{name} w kamień!"
 
-#, fuzzy
 msgid "The %{attacker}' curse falls upon the %{target}!"
-msgstr "Ta jednostka (%{name}) zostaje dotknięta klątwą mumii!"
+msgstr "%{attacker} przeklina %{name}!"
```
Comment (michkry) [inline files/lang/pl.po:582]: Here we should also change "name" for "target"

**Comment by michkry** [comment]
I've completed the revision of this PR. As for me, we can merge it into the master

**Comment by ihhub** [comment]
@Arthusppp , thank you so much for improving Polish translation of the project!

**Pull request merged**